app/models/user.rb

```diff
@@ -64,8 +64,10 @@ def confirmed?
     email_addresses.where.not(confirmed_at: nil).any?
   end
 
-  def accepted_rules_of_use?
-    self.accepted_terms_at.present?
+  def accepted_rules_of_use_still_valid?
+    if self.accepted_terms_at.present?
+      self.accepted_terms_at > IdentityConfig.store.rules_of_use_updated_at
+    end
   end
```

Comment:
Just realized, we don't have a spec for this logic in `user_spec.rb`, can we add some unit tests for it?